types/experiments.ts

```diff
@@ -0,0 +1,72 @@
+/** All fields are publically accessible via the API and available to clients */
+export interface Experiment {
+  /** Unique identifier for the experiment */
+  slug: string;
+  /** Publically-accesible name of the experiment */
+  public_name: string;
+  /** Short public description of the experiment */
+  public_description: string;
+  /** Experimenter URL */
+  experiment_url: string;
+
+  /** Is the experiment currently live in production? i.e., published to remote settings? */
+  is_published: boolean;
+  /** Are we continuing to enroll new users into the experiment? */
+  is_enrollment_paused: boolean;
```

Comment:
I think we'll need to think through what the workflow for this is.  Right now for Normandy experiments we enact this by hand but for these automated experiments, will Experimenter have to set an internal timer and update the recipe in Remote Settings to flip this bool at the correct time (which will require a manual sign off for the change)?

Reply:
Yeah, I think we better file that – at least for scheduling the pausing of enrollment